common/include/pcl/common/centroid.h

```diff
@@ -589,6 +589,61 @@ namespace pcl
     return (computeCovarianceMatrix<PointT, double> (cloud, indices, covariance_matrix));
   }
 
+
+  /** \brief Compute centroid, OBB, PCA axes of a given set of points.
+  * OBB is oriented like the three axes (major, middle and minor) with
+  * major_axis  = obb_rotational_matrix.col(0)
+  * middle_axis = obb_rotational_matrix.col(1)
+  * minor_axis  = obb_rotational_matrix.col(2)
+  * one way to visualize OBB when Scalar is float:
+  * Eigen::Vector3f position(obb_position(0), obb_position(1), obb_position(2));
+  * Eigen::Quaternionf quat(obb_rotational_matrix);
+  * viewer->addCube(position, quat, obb_dimensions(0), obb_dimensions(1), obb_dimensions(2), .....);
+  * \param[in] cloud the input point cloud
+  * \param[out] centroid the centroid of the set of points in the cloud
+  * \param[out] obb_center position of the centroid of the OBB (Oriented Bounding Box)
```

Comment:
ok, the two are the same if the cloud is centrally symmetric. 
the obb centre is the centroid of the 8 OBB vertices= centre of symmetry of the OBB (so I called it centroid, but I agree it might be misleading and needs better clarification as you wrote)